### ISSUE TITLE
Google Chrome Version 138.0.7204.50 error

### ISSUE THREAD

**Issue by harun-342** [nvaccess/nvda#18329]
### Steps to reproduce:
Open a translatable site in Chrome, try to go to translation options with f6.
### Actual behavior:

NVDA becomes unresponsive, cannot access translation options.
### Expected behavior:

NVDA should be able to go to translation options again like in previous versions of Chrome.
### NVDA logs, crash dumps, and other attachments:

[nvda.log](https://github.com/user-attachments/files/20896792/nvda.log)

[nvda-old.log](https://github.com/user-attachments/files/20896795/nvda-old.log)

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2025.1.2
#### Windows version:
Windows 11, 24 H2, 26100.4351
#### Name and version of other software used to reproduce the problem:
Chrome version 138
#### Other information about your system:
Asus, AMD Ryzen 5 7520U
#### Does the problem persist after restarting your computer?
Yes
#### Have you tried other versions of NVDA? If so, please report their behavior.
No
#### If NVDA plugins are disabled, does your problem persist?
Yes
#### Does the issue persist after running the COM Registry Fix Tool from NVDA's tools menu?
Yes

**Comment by Adriani90** [comment]
Are you using UIA for Chromium in NVDA advanced settings? Or is that set to "when necessary"?

**Comment by harun-342** [comment]
@Adriani90
NVDA advanced settings are set to default, I've never enabled any such setting. I'm sure this is after the last update Chrome received.

**Comment by hwf1324** [comment]
In my case, I cannot press F6 to focus on the default translation option that pops up. However, I can press ESC to exit the translation option, press F6 to focus on the address bar, and then find the translation and switch it.

NVDA did not freeze during the above process.

CC: @cary-rowen Can you test this?

**Comment by harun-342** [comment]
@hwf1324
I am experiencing exactly what you are experiencing. Actually, that is exactly what I was trying to say, I do not have the freezing problem either, it is just that f6 does not allow me to focus on the translation options.

**Comment by gerald-hartig** [comment]
Please raise an issue with the Google Chrome team for this.

**Comment by harun-342** [comment]
Sorry, I don't know how to report this to Google developers.

**Comment by seanbudd** [comment]
@harun-342 I found this by googling "chrome bug report" https://support.google.com/chrome/answer/95315?hl=en&co=GENIE.Platform%3DDesktop
https://issues.chromium.org/issues
